netutils/config/parser.py

```diff
@@ -1862,3 +1862,53 @@ def build_config_relationship(self) -> t.List[ConfigLine]:
                 self.config_lines.append(ConfigLine(line, ()))
 
         return self.config_lines
+
+
+class RadEtxConfigParser(BaseSpaceConfigParser):
+    """Rad ETX config parser."""
+
+    comment_chars: t.List[str] = ["#", "configure", "admin", "file"]
+    banner_start: t.List[str] = []
+
+    @property
+    def banner_end(self) -> str:
+        """Demarcate End of Banner char(s)."""
+        raise NotImplementedError("Rad ETX platform doesn't have a banner.")
+
+    def is_exit_or_exit_all(self, line: str) -> bool:
+        """Determine if line has 'exit' or 'exit all' in it.
+
+        Args:
+            line: A config line from the device.
+
+        Returns:
+            True if line has 'exit' or 'exit all', else False.
+
+        Examples:
+            >>> from netutils.config.parser import RadEtxConfigParser
+            >>> RadEtxConfigParser("config system virtual-switch").is_exit_or_exit_all("config system virtual-switch")
+            False
+            >>> RadEtxConfigParser("exit").is_exit_or_exit_all("exit")
+            True
+            >>>
+        """
+        for exit_command in ["exit", "exit all"]:
+            if line.lstrip().lower() == exit_command:
+                return True
+        return False
```

Comment:
```suggestion
        return line.lstrip().lower() in ["exit", "exit all"]
```
I think this should work

Reply:
I had some oddities with this because it seems the rad device randomly returns some unfriendly newline chars.  Let me retest again though to make sure it wasn't a copy and paste issue during sanitization of the example config.